knip.jsonc

```diff
@@ -0,0 +1,46 @@
+{
+    "workspaces": {
+        ".": {
+            // These entries are complementary to the ones found in package.json
+            "entry": [
+                "lib/rules/index.js",
+                "tools/internal-rules/index.js",
+                "tools/update-rule-types.js",
```

Comment:
```suggestion
```

This has been removed in the meantime (https://github.com/eslint/eslint/commit/66f52e276c31487424bcf54e490c4ac7ef70f77f).